src/app/util/types_stub.h

```diff
@@ -175,61 +175,22 @@ typedef struct
 /**
  * @brief The broadcast endpoint, as defined in the ZigBee spec.
  */
-#define EMBER_BROADCAST_ENDPOINT (chip::kInvalidEndpointId)
+#define MATTER_BROADCAST_ENDPOINT (chip::kInvalidEndpointId)
 
 /**
  * @brief Useful to reference a single bit of a byte.
  */
-#define EMBER_BIT(nr) (1UL << (nr)) // Unsigned avoids compiler warnings re EMBER_BIT(15)
+#define MATTER_BIT(nr) (1UL << (nr)) // Unsigned avoids compiler warnings re MATTER_BIT(15)
```

Comment:
Can this just be removed, if we are making a breaking change anyway?  It's unused.

Reply:
removed